README.md

```diff
@@ -29,13 +29,26 @@ module "subnets" {
 
 * [Basic usage](https://github.com/cisagov/distributed-subnets-tf-module/tree/develop/examples/basic_usage)
 
+## Requirements ##
+
+| Name | Version |
+|------|---------|
+| terraform | ~> 0.12.0 |
+| aws | ~> 2.0 |
+
+## Providers ##
+
+| Name | Version |
+|------|---------|
+| aws | ~> 2.0 |
+
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-------:|:--------:|
-| vpc_id | The ID of the VPC where the subnets are to be created (e.g. "vpc-0123456789abcdef0"). | string | | yes |
-| subnet_cidr_blocks | A list of the CIDR blocks associated with the individual subnets in the VPC (e.g. ["10.10.0.0/16", "10.11.0.0/16""]).  Note that the CIDR blocks in this list must be contained within the larger CIDR block associated with the VPC, and they must not overlap. | list(string) | | yes |
-| tags | Tags to apply to all AWS resources created. | map(string) | `{}` | no |
+|------|-------------|------|---------|:--------:|
+| subnet_cidr_blocks | A list of the CIDR blocks associated with the individual subnets in the VPC (e.g. ["10.10.0.0/16", "10.11.0.0/16"]).  Note that the CIDR blocks in this list must be contained within the larger CIDR block associated with the VPC, and they must not overlap. | `list(string)` | n/a | yes |
```

Comment:
Small nitpick, but the required parameters in `variables.tf` are not sorted alphabetically.

Reply:
See commit 2fa8d6f.